aderyn_core/src/detect/low/lt_instead_of_le.rs

```diff
@@ -0,0 +1,94 @@
+use std::collections::BTreeMap;
+use std::error::Error;
+
+use crate::ast::NodeID;
+
+use crate::capture;
+use crate::detect::detector::IssueDetectorNamePool;
+use crate::{
+    ast::BinaryOperation,
+    context::browser::ExtractBinaryOperations,
+    context::workspace_context::WorkspaceContext,
+    detect::detector::{IssueDetector, IssueSeverity},
+};
+use eyre::Result;
+
+// HOW TO USE THIS TEMPLATE:
+// 1. Copy this file and rename it to the snake_case version of the issue you are detecting.
+// 2. Rename the TemplateDetector struct and impl to your new issue name.
+// 3. Add this file and detector struct to the mod.rs file in the same directory.
+// 4. Implement the detect function to find instances of the issue.
+
+#[derive(Default)]
+pub struct LtInsteadOfLeDetector {
+    // Keys are: [0] source file name, [1] line number, [2] character location of node.
+    // Do not add items manually, use `capture!` to add nodes to this BTreeMap.
+    found_instances: BTreeMap<(String, usize, String), NodeID>,
+}
+
+impl IssueDetector for LtInsteadOfLeDetector {
+    fn detect(&mut self, context: &WorkspaceContext) -> Result<bool, Box<dyn Error>> {
+        for function_definition in context.function_definitions() {
```

Comment:
Hey @bowbowzai ! Thank you for the PR!

Instead of `context.function_definitions()` and then extracting the BinaryOperations, why not just grab all the BinaryOperations directly?

`context.binary_operations()` :)

Reply:
Yes, updated